internal/http/check_until_broker_ready.go

```diff
@@ -23,16 +23,16 @@ func sendReadinessRequest(n8nMainServerURI string) (*http.Response, error) {
 	return client.Do(req)
 }
 
-// CheckUntilN8nReady checks forever until the n8n main instance is ready, i.e.
+// CheckUntilBrokerReady checks forever until the task broker is ready, i.e.
```

Comment:
```suggestion
// CheckUntilBrokerReady checks forever until the task broker server is ready, i.e.
```

Reply:
I don't think we need to repeat the server everywhere. Task broker is a server